src/general/IDifficulty.cs

```diff
@@ -53,6 +53,11 @@ public interface IDifficulty : IRegistryAssignable
     ///   How intense should the fog-of-war be
     /// </summary>
     public FogOfWarMode FogOfWarMode { get; }
+
+    /// <summary>
+    ///   Whether organelle unlocks are enabled or not
```

Comment:
This should clarify that if unlocks are disabled if that means then that everything is unlocked from the start.